clients/src/main/java/org/apache/kafka/common/network/BrokerEndPoint.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.network;
```

Comment:
This is only used on the server. Could we put it under the server-common module?

Reply:
As the other class is in `clients` uses BrokerEnpoint now henc kept it in clients itself, under network. As that's the only place I found to aggregate where clients and server related classes can go.

Reply:
Moved to server-common as we have now moved to map entry as per @chia7712's comment.